### ISSUE TITLE
Python3.6

### ISSUE THREAD

**Issue by ghost** [thelahunginjeet/miprest#1]
Your code doesn't work on Python3.6; Do you have a plan to make your code running on 3.6?

**Comment by thelahunginjeet** [comment]
Yes, I really need to do that.  It's somewhat nontrivial because miprest depends on other packages that aren't 3.6 compliant either.  My student and I will be working on this in the next couple of months.
